### PR TITLE
Add new notebook to validate PR #36 for serenata-toolbox

### PR DESCRIPTION
**Pull request by luizcavalcanti**
This notebook compares both old and new version of the
Chamber's CEAP dataset. I guess the content is self-explanatory.

Datasets are found [here](https://wetransfer.com/downloads/ba2887f3fcba58df229873dbe9edd5e420170521180325/241fa3) and it validates serenata-toolbox's [PR#36](https://github.com/datasciencebr/serenata-toolbox/pull/36).

Signed-off-by: Luiz Carlos Cavalcanti <cavalcanti.luiz@gmail.com>

### CODE REVIEW

**Comment by jtemporal** [comment]
I would remove the WeTransfer link that contains the data files from the notebook since the platform only will hold the data for a number of days and link it [here](https://wetransfer.com/downloads/ba2887f3fcba58df229873dbe9edd5e420170521180325/241fa3) ;) but that's totally 💅

File: develop/2017-05-21-luizcavalcant-chamber-ceap-api-version-comparison.ipynb [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "# Comparison between Chamber of Deputies CEAP dataset 1.0 and 2.0\n",
+    "\n",
+    "This notebook compares the old Chamber's CEAP dataset (the huge XML files) with the new one (CSV by year). The mais objective of this comparison is to show we didn't lose any data on the migration from the 1.0 to the much more efficient 2.0 version of the data. This validates changes to serenata-toolbox so we can ditch 1.0 datasets for good and be prepare to their extinction by the Chamber's Open Data team.\n",
```
Comment (anaschwendler) [inline develop/2017-05-21-luizcavalcant-chamber-ceap-api-version-comparison.ipynb]: 💅 There is a typo here `The mais objective` :)

**Comment by anaschwendler** [comment]
Hi @luizcavalcanti, thanks for the contribution, nice that noticed that and can help us with that analysis :)
So, the name of the notebook has a little typo too, but besides that it is totally shippable and it is one of the most important analysis made because the data from Chamber of Deputies will change, thanks for that!

**Comment by luizcavalcanti** [comment]
HI @anaschwendler, thank you so much for your review. I'll fix it and resubmit.

**Comment by anaschwendler** [comment]
Nice @luizcavalcanti !
Thanks for fixing it, I'll give it a last look and merge it :)

Thanks for that!  🎉

**Comment by jtemporal** [comment]
@cuducos do you think we should upload the datasets used by luiz to make this notebook to the S3 for reproducibility reasons? It would be nice if people could fetch them at any given time.

**Comment by cuducos** [comment]
That's be aswesome1

**Comment by jtemporal** [comment]
Done!

**Pull request merged**